lib/radar_client.js

```diff
@@ -5,7 +5,9 @@ var MicroEE = require('microee'),
     StateMachine = require('./state.js'),
     immediate = typeof setImmediate != 'undefined' ? setImmediate :
                                     function(fn) { setTimeout(fn, 1); },
-    getClientVersion = require('./client_version.js');
+    getClientVersion = require('./client_version.js'),
+    Request = require('./message_request.js'),
```

Comment:
I would put request and response inside a folder. Also, this would reflect better what they expose (i.e. message_request.js exposes a Request, not a MessageRequest).

    Request = require('./message/request.js')